config/targets.json

```diff
@@ -5,7 +5,8 @@
       {
         "url": "https://openopps-staging.18f.gov/"
       }
-    ]
+    ],
+    "slack_channel": "openopps-platform"
```

Comment:
I feel we might irritate a lot of folks if we push these out to the specified channels while still testing. Just to be safe, could we push to `ct-bot-attack`, `cf-notifications`, and/or `compliance-toolkit`? Once we see it running well in production, we can start switching over to the actual destinations.

Thoughts?